### PR TITLE
Support for React-Native specific properties

### LINKED ISSUE #58: Support for React-Native specific properties
There are several CSS properties that are specific to React-Native that are supported by `styled-components/native`. For example the following properties are part of React-Native but appear as errors when using this plugin:

```javascript
const Main = styled.View`
  shadow-color: black;
  shadow-opacity: 0.5;
  shadow-offset: 2px 2px;
`
```

Would React-Native specific properties be something to consider supporting?

### PR DESCRIPTION
**Pull request by JinleeJeong**
- Support for React-Native specific properties

Fixes #58

### CODE REVIEW

**Comment by JinleeJeong** [comment]
Hello! I am a contributor :)

I checked it as an issue and only changed the tsconfig.json file, can you give me a comment on why the build failed?

File: tsconfig.json [hunk 1]
```diff
@@ -5,7 +5,27 @@
         "outDir": "lib",
         "rootDir": "src",
         "strict": true,
-        "declaration": true
+        "declaration": true,
```
Comment (mjbvz) [inline tsconfig.json:8]: By changing the tsconfig, you only fix `typescript-styled-plugin` inside this project. Instead we want to change the defaults that this plugin itself contributes

Comment (JinleeJeong) [reply tsconfig.json:8]: @mjbvz I completely understood!! :)  Thank you so much for the detailed review. Then, can I modify the readme? Or maybe I don't need any action.

I'll revert the commit!

Comment (mjbvz) [reply tsconfig.json]: I think we can either document this or update the defaults to include these properties

**Pull request closed without merging**